front/app/api/phases/types.ts

```diff
@@ -77,7 +79,7 @@ export interface IPhaseAttributes {
   similarity_enabled?: boolean;
   similarity_threshold_title?: number | null;
   similarity_threshold_body?: number | null;
-  user_fields_in_form?: boolean;
+  user_data_collection: UserDataCollection;
```

Comment:
Had to serialize this attribute as well, because we also need this for logged out / non-admin users for behavior in the survey form